src/main/webapp/scripts/script.js

```diff
@@ -93,6 +125,17 @@ function insertVendorInfo(container, template, vendor, isModal) {
   let salecard = vendor.saleCard;
   let prefix;
   prefix = (isModal) ? 'modal' : 'card';
+
+  if (!isModal) {
+    if (!isOpened(salecard)) {
```

Comment:
Nit: You can combine these two if statements so that you don't need to indent as much
```suggestion
  if (!isModal && !isOpened(salecard)) {
```

Reply:
For sure!